### DISCUSSION TITLE
Release of sea-orm-rocket

### DISCUSSION THREAD

**Discussion by FrTerstappen** [SeaQL/sea-orm#2428] [General]
Hello,

I noticed that the last release of sea-orm-rocket is about 10 month old.
As there are newer versions of rocket and rocket_okapi I would be interested in a release with updated dependencies.

Would you be open for a PR which updates the dependencies (and fixes what needs to be fixed)? Or is this kind of update already planned or on the way?

As this is not a bug or feature request I opted for a discussion instead of an issue. I hope that this is the right place for this request.

**Comment by tyt2y3** [comment]
Yes sure, please open a PR!

**Reply by FrTerstappen** [reply]
I created https://github.com/SeaQL/sea-orm/pull/2431

I have realized that `sea-orm-rocket` does not depend on the other crates in this repository. Is there a reason that it is included here and not in a separate repository? I think this would make working on it easier and faster as the current scripts also run the tools for all the other crates that are unrelated.
